### DISCUSSION TITLE
Dashboard interface to navigate MA

### DISCUSSION THREAD

**Discussion by prbtt** [music-assistant/support#984] [Feature requests and ideas]
https://user-images.githubusercontent.com/60021461/200225032-f9233455-6510-42f7-a78c-b85569f63457.mp4

It will be so much better in terms of user experience to be able to navigate through MA within your dashboard, similar to what I've done in the video. There are some limitations with his method that you can't see the exact song that is going ti be played.

**Comment by OzGav** [comment]
The next version of HA has a new UI although i am not sure if it will be like what you are suggesting here. Please hold onto this idea and bring it up again in December after you see the new version.
